playbook/playbook.go

```diff
@@ -96,9 +96,56 @@ func validateAndSavePlaybookRun(ctx context.Context, playbook *models.Playbook,
 		return nil, fmt.Errorf("failed to create playbook run: %v", err)
 	}
 
+	if err := saveRunAsConfigChange(ctx, playbook, run, req.Params); err != nil {
+		ctx.Logger.Errorf("failed to save playbook run as config change: %v", err)
+	}
+
 	return &run, nil
 }
 
+func saveRunAsConfigChange(ctx context.Context, playbook *models.Playbook, run models.PlaybookRun, parameters any) error {
+	if run.ConfigID == nil {
+		return nil
+	}
+
+	change := models.ConfigChange{
+		ExternalChangeId: uuid.NewString(),
+		Severity:         models.SeverityInfo,
+		ConfigID:         run.ConfigID.String(),
+		ChangeType:       string(run.Status),
+		Source:           "Playbook",
+		Summary:          fmt.Sprintf("Playbook: %s", playbook.Name),
+	}
+
+	switch run.Status {
+	case models.PlaybookRunStatusScheduled:
```

Comment:
Can the Change type be `PlaybookScheduled` ?

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/user-attachments/assets/599c7f65-ff64-4886-b75c-0f195a4bdf6f)